elements/pfe-button/README.md

```diff
@@ -14,14 +14,6 @@ the only light DOM child of `pfe-button`.
 </pfe-button>
 ```
 
-### Size
-
-```html
-<pfe-button size="large">
-  <button>Large Button</button>
-</pfe-button>
-```
-
 ### Accessibility
 `<pfe-button>` addresses the issues associated with typical implementations of
 web component buttons. When using a `<pfe-button>` in a `<form>` element, the
```

Comment:
I'm still not super clear on why we're copying the button into Shadow DOM instead of just using the slotted element. If like to get some regression tests out together so we can evaluate just using slot.

Reply:
I added some code to synchronize the `type` attr. I'd prefer to either use the slotted button or the ElementInternals polyfill, but that will have to wait until after the break. I also added your demo cases. Thanks!